recipe/meta.yaml

```diff
@@ -46,6 +46,7 @@ test:
     {% endfor %}
     - test -f ${PREFIX}/include/bzlib.h  # [unix]
     - test -f ${PREFIX}/lib/libbz2.a  # [unix]
+    - test -f ${PREFIX}/lib/libbz2.so.1.0  # [unix]
```

Comment:
Hm, it seems the `Makefile-libbz2_so` is Linux-specific. Travis fails with:
```
+ make -f Makefile-libbz2_so 'CFLAGS= -arch x86_64 -mmacosx-version-min=10.9 -m64 -Wall -Winline -O2 -g -D_FILE_OFFSET_BITS=64 -fPIC'

gcc -shared -Wl,-soname -Wl,libbz2.so.1.0 -o libbz2.so.1.0.6 blocksort.o huffman.o crctable.o randtable.o compress.o decompress.o bzlib.o

ld: unknown option: -soname
```
I found [a fork of bzip2 that fixes this](https://github.com/centricular/bzip2), but I'm not sure I'm ready to go there since my OS X knowledge is limited.

Instead, would it be ok to not generate the shared library on OS X?